### ISSUE TITLE
Serialization error when enqueueing

### ISSUE THREAD

**Issue by brandonhilkert** [phstc/shoryuken#549]
New to Shoryuken (so it's probably user). I a large Sidekiq cluster and i'm interested in moving some of the work to SQS due to load.

Here's the basics of my job:

```ruby
module Pollers
  class GoogleDrivePollWorker
    include Shoryuken::Worker
    shoryuken_options queue: "poller", auto_delete: true

    perform(sqs_msg, connection_id)
       ...
    end
end
```

```
irb(main):001:0> Pollers::GoogleDrivePollWorker.perform_async Connection.last.id
  Connection Load (2.8ms)  SELECT  "connections".* FROM "connections" ORDER BY "connections"."id" DESC LIMIT $1  [["LIMIT", 1]]
Traceback (most recent call last):
        1: from (irb):1
ArgumentError (expected params[:message_body] to be a String, got value 861969468 (class: Integer) instead.)
```

If you have a chance to let me know what I'm doing wrong, that'd be 👌

**Comment by phstc** [comment]
Hi @brandonhilkert 

 tl;dr use `Connection.last.id.to_s`

When you call `perform_async` Shoryuken calls `send_message` from the aws-sdk and it requires the body to be a string.

**Comment by brandonhilkert** [comment]
Got it. Thanks for the quick reply. Can you point me to the documentation for handling of multiple arguments?

```
    def perform(sqs_msg, connection_id, msg)
```
```
irb(main):001:0> Pollers::GoogleDrivePollWorker.perform_async Connection.last.id.to_s, "test"
  Connection Load (1.2ms)  SELECT  "connections".* FROM "connections" ORDER BY "connections"."id" DESC LIMIT $1  [["LIMIT", 1]]
Traceback (most recent call last):
        1: from (irb):1
TypeError (no implicit conversion of Symbol into Integer)
```

**Comment by brandonhilkert** [comment]
Nm. I think I [found my answer](https://github.com/phstc/shoryuken/wiki/Sending-a-message#using-perform_async).

**Comment by phstc** [comment]
@brandonhilkert Standard workers don't support multiple arguments. One option for sending multiple arguments is to send them as a Hash.

```ruby
Pollers::GoogleDrivePollWorker.perform_async(id: Connection.last.id.to_s, msg: "test")
```

See https://github.com/phstc/shoryuken/wiki/Sending-a-message#using-perform_async

If you use [Shoryuken with Active Job](https://github.com/phstc/shoryuken/wiki/Rails-Integration-Active-Job), you can send multiple arguments as follows [`GuestsCleanupJob.perform_later(guest1, guest2, filter: 'some_filter')`](https://edgeguides.rubyonrails.org/active_job_basics.html#enqueue-the-job).
